testutil/tx/signer.go

```diff
@@ -57,7 +56,7 @@ func NewSigner(sk cryptotypes.PrivKey) keyring.Signer {
 }
 
 // Sign signs the message using the underlying private key
-func (s Signer) Sign(_ string, msg []byte, signMode signing.SignMode) ([]byte, cryptotypes.PubKey, error) {
+func (s Signer) Sign(_ string, msg []byte, _ signing.SignMode) ([]byte, cryptotypes.PubKey, error) {
```

Comment:
Out of three input this function is only using one. I imagine there is a reason to remove them. What is this reason?